apps/extension/scripts/kernel/handlers/sign.ts

```diff
@@ -1,19 +1,71 @@
+import { Keypair, VersionedTransaction } from '@solana/web3.js';
 import { decryptWithPasscode } from '@walless/crypto';
 import { MessengerCallback } from '@walless/messaging';
+import { signMessage } from '@walless/network/src/solana/transactions';
```

Comment:
should be
```typescript
import { signMessage } from '@walless/network';
```
make sure to export `signMessage` from `@walless/network` package's index file.